Kentico.Kontent.Delivery/Configuration/DeliveryOptions .cs

```diff
@@ -61,5 +61,11 @@ public class DeliveryOptions
         /// Gets or sets configuration of the default retry policy.
         /// </summary>
         public DefaultRetryPolicyOptions DefaultRetryPolicyOptions { get; set; } = new DefaultRetryPolicyOptions();
+
+        /// <summary>
+        /// Gets or sets a value that determines if the client provides total count for each query to items.
```

Comment:
Gets or sets a value that determines if the client includes the total number of items matching the search criteria in the response. This behavior can also be enabled for individual requests with the <see cref="IncludeTotalCountParameter"/>.